### ISSUE TITLE
Heroku Support

### ISSUE THREAD

**Issue by besoeasy** [Seuntjie900/DiceBot#58]
Can You Port This To A Supported Minimal Language that is support by heroku ? like python or node js
only need programming mode, so definitely can run on cloud 24*7

i already donated you many times, if this is possible  let me know i will send few $$ for trouble

**Comment by Seuntjie900** [comment]
Nope, sorry. You also probably shouldn't be running a gambling bot 24/7, it'll eat up all of your money.

The bot is being rewritten in .net core, so it should be able to run on linux soon, and there will be a console version of it. won't be able to run on heroku, but heroku isn't the only cloud platform out there.

**Comment by besoeasy** [comment]
understand i was .NET programmer for years, i understand !

anyways i be working on node js version of your bot !

thanks for reply
